

-------------------------------------------------------------------------
Systeminfo:
CPU:               Intel64 Family 6 Model 158 Stepping 9, GenuineIntel
Number of CPUs:    8
RAM:               31.9GB
Diskinfo:          26.6GB/446.5GB
Operating system:  Windows-10-10.0.19045
Long path support: False
-------------------------------------------------------------------------

= 30.10.2024 16:20:58 (UTC +0:00) === ecu.test 2024.3.0.156114+c108ea21784a 156114+c108ea21784a (2024/09/19 12:15:31) (64bit) ===
Python 3.11.9 (tags/v3.11.9:de54cf5, Apr  2 2024, 10:12:12) [MSC v.1938 64 bit (AMD64)] / wx: 4.2.1+tt.5
Protocol: log level Default. Additional logging active: default output

License: ecu.test
         - License options:
             + ecu.test diagnostics
         - valid up to version: 2024.4
         - valid until: 2025-01-28
         - licensed for: Mahindra_Racing_UK
         - runner: False
         - Server: license01.tracetronic.de



2024-10-30 16:20:58.810 9080 ManagedWorker-3*PE.Run() DEBUG_EXC_PRINT:
                        Traceback (most recent call last):
                          File "tts\core\package\engine\ExecutionMappingSpaceManager.py", line 558, in __CallRegisterXaObjects
                          File "tts\testModule\bus\mapping\XABusSignalVariableExec.py", line 24, in PrepareWrite
                          File "tts\core\mapping\xa\XAVariableExec.py", line 88, in GetPort
                          File "tts\core\mapping\xa\XAVariableExec.py", line 69, in GetPortId
                        tt._Exceptions_.TTError: Missing port assignment for BUSACCESS(CAN1, AccessManipulation)
2024-10-30 16:20:58.833 9080 ManagedWorker-3*PE.Run() DEBUG_EXC_PRINT:
                        Traceback (most recent call last):
                          File "tts\core\package\engine\ExecutionMappingSpaceManager.py", line 558, in __CallRegisterXaObjects
                          File "tts\testModule\bus\mapping\XABusSignalVariableExec.py", line 33, in PrepareStimulus
                          File "tts\core\mapping\xa\XAVariableExec.py", line 88, in GetPort
                          File "tts\core\mapping\xa\XAVariableExec.py", line 69, in GetPortId
                        tt._Exceptions_.TTError: Missing port assignment for BUSACCESS(CAN1, AccessManipulation)
